egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -64,7 +64,7 @@ where "nvcc" is installed.
 EOF
 fi
 
-use_delay=false
+use_delay=
 if [ $label_delay -gt 0 ]; then use_delay=true; fi
```

Comment:
It seems like the use_delay variable is not really necessary at all now.
Couldn't you do something like
if [ $label_delay -gt 0 ]; then dir=${dir}_ld$label_delay; fi
... then there would be no need for the use_delay variable which IMO just confuses things.